TestEnvironmentUsage.md

```diff
@@ -0,0 +1,55 @@
+# Test Environment Usage
+
+See [Test Environment Setup](TestEnvironmentSetup.md) for details on setting up the test environment.
+
+### Using Meld
+
+Navigate to https://meld.interop.community/REMS/apps to access the Meld  sandbox.  There are several things to be familiar with.
+
+![](./test-environment-images/meld_apps.png)
+
+These are the registered apps.  A registered app consists of:
+-	A launch URI, which is the actual SMART app
+-	A redirect URI, which is a valid location for the SMART app to redirect
+-	Scopes, which is the resources we have access to
+After registering an app, we get a client id, which is the client id of the SMART app.
+
+The SMART app must be made aware of the client id in order to launch correctly. In the case of the REMS SMART app, registration is handled at the time of launch if the app doesn’t already know which client id to use, which should make the process seamless.  
+
+When launching an app, you are connecting it to Meld’s EHR system.
+I’ve already uploaded our default patient, Jon Snow, and all of his related resources.  But to upload more resources to Meld, follow these steps:
+
+-	All references to other resources must be correctly made with regard to assigned ID by the EHR.  
+    - In other words, if you want to upload a resource A that references another resource B, you will first have to upload B, at which point the EHR will assign it an ID.  Then you can take that ID and use it in A as a reference.
+-	Get an access token
+    - There are several ways to do this.  The easiest method is to just launch a smart app and use the access token it creates.
```

Comment:
maybe we could have a short blurb somewhere about how to find the access token a SMART app creates. But this feels a bit hacky. It might be better to just suggest that users just use the built in data manager on meld

Reply:
@KeeyanGhoreshi, I believe that this is the document that you had originally written. Is the access token being referred to here the one needed to query data from the EHR?

Reply:
yes, I guess I don't remember if there's a reason I didn't originally write that we should use the data manager.

Reply:
I just pushed a change to point people to the setup document that describes this.